src/reference/modules/usage/pages/advanced_node_management.adoc

```diff
@@ -452,6 +452,7 @@ image::node-lifecycle-settings.png[]
 In the future, these states will be configurable on a per node basis at acceptation, and the
 lifecycle states list will be configurable by users.
 
+[[disable_listen_port_5309]]
```

Comment:
is it not a problem that there is no space between this line and the one below ?

Reply:
no, for reference it's done like that elsewhere (and I see that the two of us are very fluent in doc :)) )

Reply:
totally fluent, and i would have done like you, copy from other file :)